pkgs/by-name/pa/paretosecurity/package.nix

```diff
@@ -17,17 +17,18 @@ buildGoModule (finalAttrs: {
     webkitgtk_4_1
   ];
   pname = "paretosecurity";
-  version = "0.1.9";
+  version = "0.2.13";
 
   src = fetchFromGitHub {
     owner = "ParetoSecurity";
     repo = "agent";
     rev = finalAttrs.version;
-    hash = "sha256-KJs4xC3EtGG4116UE+oIEwAMcuDWIm9gqgZY+Bv14ac=";
+    hash = "sha256-G2An1Br/NJcIEKaM/pPEGkJazD9UFypRZ/KLmqkNUTo=";
   };
 
-  vendorHash = "sha256-3plpvwLe32AsGuVzdM2fSmTPkKwRFmhi651NEIRdOxw=";
+  vendorHash = "sha256-YnyACP/hJYxi4AWMwr0We4YUTbWwahKAIYN6RnHmzls=";
   proxyVendor = true;
+  subPackages = ["cmd/paretosecurity"];
```

Comment:
This is needed to prevent nix from building the Windows agent (that is in the same repo).